torch/_inductor/ir.py

```diff
@@ -2917,45 +2917,41 @@ def create(cls, kernel, *args, **kwargs):
                 unflatten_args,
             ) = cls.process_kernel(kernel, *args, **kwargs)
 
-        if isinstance(example_output, (list, tuple)):
-            packed = FallbackKernel(
-                MultiOutputLayout(tensor_args[0].get_device()),
-                kernel,
-                tensor_args,
-                non_tensor_args,
-                unflatten_args,
-            )
-            return [
-                (
+        def generate_output(output, index=""):
```

Comment:
could add asserts that the length of the output at runtime is the same as we observed here ?

Reply:
We do have `assert_size_stride` generated and that will trigger an issue if the output indexing is OOB (that's how I debugged one test failure), so I guess I will skip adding another assert for now.

Reply:
that'll catch when the list runtime list is smaller but it won't catch when the runtime list is larger